### ISSUE TITLE
[Bug]: Running operator failed for reporting lots of modules are not found in clean env

### ISSUE THREAD

**Issue by binbinlv** [towhee-io/towhee#1450]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Running operator failed for reporting lots of modules are not found, like:
pkg_resources.DistributionNotFound: The 'av' distribution was not found and is required by the application
pkg_resources.DistributionNotFound: The 'opencv-python' distribution was not found and is required by the application
......

### Expected Behavior

No errors

### Steps To Reproduce

For example: Running the following script
```python
import logging
import os
import time
import subprocess
from towhee import ops

logger = logging.getLogger()
log_path = 'action_classification.log'

# Test action-classification operators
action_classification = {
    'pytorchvideo': [
        {'model_name': 'c2d_r50'},
        {'model_name': 'i3d_r50'},
        {'model_name': 'slow_r50'},
        {'model_name': 'slowfast_r50'},
        {'model_name': 'slowfast_r101'},
        {'model_name': 'x3d_xs'},
        {'model_name': 'x3d_s'},
        {'model_name': 'x3d_m'},
        {'model_name': 'mvit_base_16x4'},
        {'model_name': 'mvit_base_32x3'},
    ],
    'actionclip': [
        {'model_name': 'clip_vit_b16'},
        {'model_name': 'clip_vit_b32'}
    ],
    'timesformer': [
        {'model_name': 'timesformer_k400_8x224'}
    ]
}


def test(data_path):
    if not os.path.isfile(data_path):
        subprocess.call(
            f'wget -O {data_path} https://dl.fbaipublicfiles.com/pytorchvideo/projects/archery.mp4',
            shell=True
        )

    video_decoder = ops.video_decode.ffmpeg()
    # Since the video_decoder returns generator, need to extract output from generator for repetitive usage
    input_data = [x for x in video_decoder(data_path)]
    handler = logging.FileHandler(log_path)
    logger.addHandler(handler)

    for k, v in action_classification.items():
        init_op = getattr(ops.action_classification, k)
        for args in v:
            try:
                op = init_op(**args)
                outs = op(input_data)
                assert outs[0][0] == 'archery'
                assert len(outs[1]) == 5
                assert outs[2].shape[0] > 0
                print('Success:', k, str(args))
                logger.info('Success %s - %s', k, str(args))
            except Exception as e:
                logger.error('Fail to run %s - %s', k, str(args))
                print(f'ERROR MESSAGE: {e}')
                pass
            time.sleep(5)
    print('Finished.')

if __name__ == '__main__':
    test('./archery.mp4')
```

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93): latest pre-0.7
- OS(Ubuntu or CentOS):
- CPU/Memory:
- GPU:
- Others:
```

**Comment by binbinlv** [comment]
I think this maybe for the fix for issue #1433.

**Comment by binbinlv** [comment]
Test it again, it is good in clean python env and clean cache. And this issue will occur when the cache for python3.8 is not be cleaned and when installing towhee in python 3.7,  this is compatibility issue which will be fixed in next release.

And the workaround is clean the cache when you want to use new python version and new env.
